src/drt/README.md

```diff
@@ -17,15 +17,173 @@ guide format.
 
 ## Commands
 
-## Report wire length
-
-Check the [global router README](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/grt/README.md)
-file for the full description of the `report_wire_length` command.
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
+
+### Detailed Route
+
+```tcl
+detailed_route 
+    [-output_maze filename]
+    [-output_drc filename]
+    [-output_cmap filename]
+    [-output_guide_coverage filename]
+    [-drc_report_iter_step step]
+    [-db_process_node name]
+    [-disable_via_gen]
+    [-droute_end_iter iter]
+    [-via_in_pin_bottom_layer layer]
+    [-via_in_pin_top_layer layer]
+    [-or_seed seed]
+    [-or_k_ k]
+    [-bottom_routing_layer layer]
+    [-top_routing_layer layer]
+    [-verbose level]
+    [-distributed]
+    [-remote_host rhost]
+    [-remote_port rport]
+    [-shared_volume vol]
+    [-cloud_size sz]
+    [-clean_patches]
+    [-no_pin_access]
+    [-min_access_points count]
+    [-save_guide_updates]
+    [-repair_pdn_vias layer]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-output_maze` | Path to output maze log file (e.g. `.output.maze.log`). |
+| `-output_drc` | Path to output drc report file (e.g. `.output.drc.rpt`). |
+| `-output_cmap` | Path to output congestion map file (e.g. `.output.cmap`). |
+| `-output_guide_coverage` | Path to output guide coverage file (e.g. `_sample_coverage.csv`). |
+| `-drc_report_iter_step` | Report drc on each iteration which is a multiple of this step (default 0, integer). |
+| `-db_process_node` | Specify the process node. |
+| `-disable_via_gen` | Option to diable via generation with bottom and top routing layer. (default not set) | 
+| `-droute_end_iter` | Number of detailed routing iterations (must be a positive integer <= 64). |
+| `-via_in_pin_bottom_layer` | Via-in pin bottom layer name. |
+| `-via_in_pin_top_layer` | Via-in pin top layer name. |
+| `-or_seed` | Random seed for the order of nets to reroute (default -1, integer). | 
+| `-or_k` | Number of swaps is given by $k * sizeof(rerouteNets)$ (default 0, integer). |
+| `-bottom_routing_layer` | Bottommost routing layer name. |
+| `-top_routing_layer` | Topmost routing layer name. |
+| `-verbose` | Sets verbose mode if the value is greater than 1, else non-verbose mode (must be integer, or error will be triggered.) |
+| `-distributed` | Enable distributed mode with Kubernetes and Google Cloud, [guide](./doc/Distributed.md). |
+| `-remote_host` | The host IP. |
+| `-remote_port` | The value of the port to access from. |
+| `-shared_volume` | The mount path of the nfs shared folder. |
+| `-cloud_size` | The number of workers. |
```

Comment:
Let's group the `-distributed` options in a separated table. A note about the need for additional setup is required.